test/Sentry.Tests/Protocol/DsnTests.cs

```diff
@@ -133,21 +126,15 @@ public void Ctor_NullDsn_ThrowsArgumentNull()
     }
 
     [Fact]
-    public void TryParse_SampleValidDsnWithoutSecret_Succeeds()
-    {
-        Assert.NotNull(Dsn.TryParse(DsnSamples.ValidDsnWithoutSecret));
-    }
-
-    [Fact]
-    public void TryParse_SampleValidDsnWithSecret_Succeeds()
```

Comment:
Shouldn't we still keep the Parse and TryParse tests? Though we don't use the secret ourself, it's still valid input from the user.

Reply:
@bruno-garcia is this correct. from your statement that spured this, i cant tell if secret is still valid?

Reply:
we don't need to validate secret anymore. Test can be deleted.

Reply:
re added the tests for with secret